hr_employee_age/models/hr_employee.py

```diff
@@ -8,7 +8,8 @@
 class HrEmployee(models.Model):
     _inherit = "hr.employee"
 
-    age = fields.Integer(compute="_compute_age")
+    # Add groups="hr.group_hr_user" to age field according to birthday field definition
```

Comment:
This is not adding value as explanation. It doesn't say the why, just the what, which is pretty obvious reading the code.

Reply:
Comment modified.

Reply:
OK, thank you. Maybe is now too much verbose, as it can be put simply as `# Group is needed as the dependency field `birthday` already has it`, but no special problem in being longer.